percona/version/version.go

```diff
@@ -71,7 +66,7 @@ func fetchVersions(ctx context.Context, endpoint string, vm Meta) (DepVersion, e
 		HelmDeployCr:       &vm.HelmDeployCR,
 		HelmDeployOperator: &vm.HelmDeployOperator,
 		SidecarsUsed:       &vm.SidecarsUsed,
-		Product:            ProductName,
+		Product:            v2beta1.ProductName,
```

Comment:
I think it will be v2. @egegunes should we wait and then update it?

Reply:
Yes, we should.